### PR TITLE
Bug/readmes

### PR DESCRIPTION
**Pull request by jmccreight**
From the commit: 

```
Documentation: 
* minor corrections, clarifications
* some cross-referencing (DEVELOPER <> CONTRIBUTING & doxygen <> rtd)
* suggestion of .vscode/settings_template.json for vsc newbs
```

### CODE REVIEW

**Comment by jmccreight** [comment]
This is the look of the new "MAINPAGE.md" for the doxygen output
Screen Shot 2022-01-20 at 2 12 05 PM

**Review by jdhughes-usgs — APPROVED** [review]

File: .build_rtd_docs/Doxyfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ PROJECT_NAME           = "MODFLOW 6"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "version 6.2.2"
+PROJECT_NUMBER         = "version 6.3.0"
```
Comment (jmccreight) [inline .build_rtd_docs/Doxyfile:40]: this should probably be fixed?

File: .doc/requirements.txt [hunk 2]
```diff
@@ -3,6 +3,4 @@ ipython
 ipykernel
 rtds_action
 sphinx_rtd_theme
-
-
-
+recommonmark
```
Comment (jmccreight) [inline .doc/requirements.txt:6]: i had to manually add this... not sure why CI dosent need it

File: DEVELOPER.md [hunk 3]
```diff
@@ -59,10 +59,13 @@ Then install all packages necessary to run the tests either by executing [instal
 
 Intel fortran can be used to compile MODFLOW 6 and associated utilities and generate distributable files (if not using gfortran).
 Download the Intel oneAPI HPC Toolkit: https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit/download.html
+Documentation describing how to set the intel envrionment variables can be found [here](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup.html).
 
 ### LaTeX (optional)
 
 [LaTeX](https://www.latex-project.org/) which is used to generate the MODFLOW 6 release notes and Input/Output documents (docs/mf6io/mf6io.nightlybuild).
+JLM: should this be more specific? I even believe the installs are in .github/workflow
+JLM: what about doxygen+graphivz? looks like only linux installs
```
Comment (jmccreight) [inline DEVELOPER.md:68]: This should be removed

**Comment by jmccreight** [comment]
I'm not sure why this looks so bad locally, but the new first line of the contents is just a link to the doxygen docs.
Screen Shot 2022-01-20 at 2 46 51 PM

File: DEVELOPER.md (hunk 3 above)
Comment (jmccreight) [reply DEVELOPER.md:68]: I did doxygen and graphviz via conda.

File: .build_rtd_docs/Doxyfile (hunk 1 above)
Comment (jdhughes-usgs) [reply .build_rtd_docs/Doxyfile:40]: That is updated by one of our scripts when doxygen is built on CI. But should update at some point so now is good.

File: .doc/requirements.txt (hunk 2 above)
Comment (jdhughes-usgs) [reply .doc/requirements.txt:6]: I don't know why it doesn't cause a problem either

File: DEVELOPER.md (hunk 3 above)
Comment (jdhughes-usgs) [reply DEVELOPER.md:68]: I think I may have used macports for these

**Comment by jdhughes-usgs** [comment]
I don't know why either. I saw this last time I rebuilt locally.

**Review by langevin-usgs — APPROVED** [review]
Nice additions!

**Pull request merged**